### PR TITLE
Closes #1256: Update ic_device_desktop

### LINKED ISSUE #1256: Update ic_device_desktop to match photon icon
Photon icon: https://github.com/FirefoxUX/photon-icons/blob/master/icons/android/device-desktop-24.xml

### PR DESCRIPTION
**Pull request by dnarcese**

### CODE REVIEW

File: components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml [hunk 1]
```diff
@@ -3,11 +3,12 @@
...
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="16dp"
-    android:height="16dp"
-    android:viewportHeight="16"
-    android:viewportWidth="16">
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
```
Comment (pocmo) [inline components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml]: This icon has a different size. Is this intentional?

File: components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml [hunk 2]
```diff
@@ -3,11 +3,12 @@
...
   <path
       android:fillColor="#FFFFFF"
-      android:pathData="M15.5,12L15,12L15,3a1,1 0,0 0,-1 -1L2,2a1,1 0,0 0,-1 1v9L0.5,12a0.5,0.5 0,0 0,-0.5 0.5v1a0.5,0.5 0,0 0,0.5 0.5h15a0.5,0.5 0,0 0,0.5 -0.5v-1a0.5,0.5 0,0 0,-0.5 -0.5zM10,13L6,13v-1h4zM13,11L3,11L3,4h10z" />
+      android:fillType="evenOdd"
```
Comment (pocmo) [inline components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml]: Wasn't there an issue with "evenOdd" not being supported on all versions of Android?

File: components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml (hunk 1 above)
Comment (dnarcese) [reply components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml]: Yes, I needed the size to match the other icons FFTV is using (which are all 24x24).

Comment (pocmo) [reply components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml]: Oh, I just noticed that this was 16dp **before**. Yep, makes sense!

File: components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml (hunk 2 above)
Comment (dnarcese) [reply components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml]: I regenerated the SVG so it wouldn't use `evenOdd` 👍

Comment (pocmo) [reply components/ui/icons/src/main/res/drawable/mozac_ic_device_desktop.xml]: Awesome. Thank you! I was trying to find any information on whether the support library can handle "evenOdd". The system will only be able to load it with API 24+ but maybe the support library can do it. Anyhow, now we can just merge it. :)

**Pull request merged**